conf/logback.xml

```diff
@@ -43,27 +43,24 @@
         <appender-ref ref="STDOUT_IGNORE_NETTY" />
     </logger>
 
-    <logger name="com.ning.http.client" level="WARN"/>
-    <logger name="org.asynchttpclient.netty.channel.DefaultChannelPool" level="WARN"/>
-    <logger name="play.shaded.ahc.org.asynchttpclient.netty.channel.DefaultChannelPool" level="WARN"/>
-    <logger name="play.core.netty.utils.ServerCookieDecoder" level="WARN"/>
-    <logger name="play.shaded.ahc.org.asynchttpclient.netty.request.NettyRequestSender" level="WARN"/>
-    <logger name="play.shaded.ahc.org.asynchttpclient.netty.handler.HttpHandler" level="WARN"/>
-    <logger name="play.shaded.ahc.org.asynchttpclient.netty.channel.ChannelManager" level="WARN"/>
-    <logger name="org.asynchttpclient.netty" level="WARN"/>
+    <logger name="com.google.inject" level="WARN"/>
```

Comment:
Why are we adding this specifically - not mentioned in acceptance criteria and haven't seen it in other repos we have changed recently?

Reply:
So I added this as was in the example for step 2 for the logback file https://github.com/hmrc/vat-summary-frontend/blob/master/conf/logback.xml though I may have misinterpreted which needed adding removing there.

Reply:
Happy to leave in as it is present in other repos and is not affecting functionality